packages/react-native-editor/src/index.js

```diff
@@ -25,6 +25,10 @@ const reactNativeSetup = () => {
 	// eslint-disable-next-line no-console
 	console.disableYellowBox = true;
 
+	if ( UIManager.setLayoutAnimationEnabledExperimental ) {
+		UIManager.setLayoutAnimationEnabledExperimental( true );
+	}
+
```

Comment:
I saw that we also enable this [here](https://github.com/WordPress/gutenberg/blob/5e7afd6c2b9638a3ce180fdc50d90f17ea1217a7/packages/components/src/mobile/bottom-sheet/bottom-sheet-context.native.js#L10-L17), probably that was done because we only had layout animations in the Bottom sheet component but now that we'll use it in more places, I think it makes more sense to only keep this one, wdyt?

Reply:
Makes sense! Removed the duplication with https://github.com/WordPress/gutenberg/pull/32489/commits/f1789e223a96ae681e5840577a2edfdb718f170f.